src/main/java/com/rackspacecloud/metrics/ingestionservice/influxdb/providers/CreateInfluxDB.java

```diff
@@ -0,0 +1,53 @@
+package com.rackspacecloud.metrics.ingestionservice.influxdb.providers;
+
+import org.influxdb.InfluxDBFactory;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.config.BackupProperties;
+import org.influxdb.InfluxDB;
+import org.influxdb.dto.Query;
+import org.influxdb.dto.Pong;
+import org.influxdb.dto.QueryResult;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.providers.RestoreDBInfluxDB;
+import java.io.IOException;
+import com.rackspacecloud.metrics.ingestionservice.influxdb.providers.NotcreatedDBInfluxDB;
+//import org.springframework.kafka.test.context.EmbeddedKafka;
+//@ActiveProfiles(value = { "test" })
+//@EmbeddedKafka(partitions = 1, topics = { IngestionServiceApplicationTests.UNIFIED_METRICS_TOPIC })
+@EnableConfigurationProperties(BackupProperties.class)
+public class CreateInfluxDB {
+    // User name
+    private String username;
+    // Password
+    private String password;
+    // Connection address
+    private String openurl;
+    // data base
+    private String database;
+    // Reservation strategy
+    private String retentionPolicy;
+    private InfluxDB influxDB;
+
+    public CreateInfluxDB(String username, String password, String openurl, String database,
+                          String retentionPolicy) {
+        this.username = "ravi5626";
+        this.password = "*****";
```

Comment:
Literal strings throughout, especially your username need to be properly parameterized. It can't be expected the code be modified for every test run. In Spring Boot you can externalize configuration from yaml/property files and then access from POJOs, [as described here](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-typesafe-configuration-properties).